js/index.js

```diff
@@ -141,52 +140,18 @@ Raven.context(async () => {
     document.getElementById("app").style.visibility = "hidden";
     return;
   }
-  const __extraWindows = [];
-  let __initialWindowLayout = null;
 
+  let butterchurnConfig = null;
   if (isButterchurnSupported()) {
     const startWithMilkdropHidden =
       document.body.clientWidth < MIN_MILKDROP_WIDTH ||
       (!library && skinUrl != null) ||
       screenshot;
 
-    __extraWindows.push({
-      id: "milkdrop",
-      title: "Milkdrop",
-      isVisualizer: true,
-      Component: MilkdropWindow,
-      open: !library && !startWithMilkdropHidden
-    });
-
-    if (startWithMilkdropHidden) {
-      __initialWindowLayout = {
-        [WINDOWS.MAIN]: { position: { x: 0, y: 0 } },
-        [WINDOWS.EQUALIZER]: { position: { x: 0, y: 116 } },
-        [WINDOWS.PLAYLIST]: { position: { x: 0, y: 232 }, size: [0, 0] },
-        milkdrop: { position: { x: 0, y: 348 }, size: [0, 0] }
-      };
-      if (library) {
-        __initialWindowLayout[WINDOWS.MEDIA_LIBRARY] = {
-          position: { x: 0, y: 348 },
-          size: [0, 0]
-        };
-      }
-    } else {
-      __initialWindowLayout = {
-        [WINDOWS.MAIN]: { position: { x: 0, y: 0 } },
-        [WINDOWS.EQUALIZER]: { position: { x: 0, y: 116 } },
-        [WINDOWS.PLAYLIST]: { position: { x: 0, y: 232 }, size: [0, 4] },
-        milkdrop: { position: { x: 275, y: 0 }, size: [7, 12] }
-      };
-      if (library) {
-        __initialWindowLayout[WINDOWS.MEDIA_LIBRARY] = {
-          position: { x: 275, y: 0 },
-          size: [7, 12]
-        };
-      }
-    }
```

Comment:
I think the position of the windows should still be defined here. I'd prefer the initial Layout be at discretion of the site that consumes Webamp. I know `__initialWindowLayout` is not a public option now, but I think it should be when we add Butterchurn.

Reply:
oh that makes sense, i'll move it back